napi/oxlint2/test/e2e.test.ts

```diff
@@ -0,0 +1,21 @@
+import { execa } from 'execa';
+import { describe, expect, it } from 'vitest';
+
+function normalizeOutput(output: string): string {
+  return output.replace(/Finished in \dms/, 'Finished in Xms');
+}
+
+describe('cli options for bundling', () => {
+  it('should lint a directory', async () => {
+    const { stdout, exitCode } = await execa(
+      'node',
+      ['../../../src/index.js'],
+      {
+        cwd: 'test/fixtures/basic',
+      },
```

Comment:
not a massive fan of these hard-coded ish args. Rolldown somehow injects `rolldown` into the path so it doesn't have to do this 🤔

Reply:
When we add more tests, we could have a reusable function to run `oxlint2`. e.g.:

```js
const PACKAGE_ROOT_PATH = path.dirname(import.meta.dirname),
  ENTRY_POINT_PATH = path.join(PACKAGE_ROOT_PATH, 'index.js');

async function runOxlint(cwd, args = []) {
  return await execa(
    'node',
    [ ENTRY_POINT_PATH, ...args ],
    { cwd: path.join(PACKAGE_ROOT_PATH, cwd) },
  );
}

const { stdout, exitCode } = await runOxlint('test/fixtures/basic');
```

That's not quite as nice as Rolldown's solution, but it's simpler.